src/include/miopen/conv/invokers/impl_gemm_dynamic.hpp

```diff
@@ -35,6 +35,41 @@
 namespace miopen {
 namespace conv {
 
+namespace igemm_dynamic {
+template <typename T>
+T gcd(T x, T y)
```

Comment:
C++17 supports gcd function. So, we need until we upgrade the compiler version. Could you please move this function to some header file include/miopen so that it could be used by other files not just impl_gemm_dynamic.

Reply:
I move this into `src/include/miopen/numeric.hpp`